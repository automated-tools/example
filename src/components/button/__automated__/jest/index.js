import React from 'react';
import renderer from 'react-test-renderer';

import { Component, title } from '../';

it(title, () => {
  const tree = renderer.create(<Component label="test" />).toJSON();
  expect(tree).toMatchSnapshot();
});
