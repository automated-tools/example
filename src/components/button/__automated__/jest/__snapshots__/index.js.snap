// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hello 1`] = `
<button
  style={
    Object {
      "fontSize": 20,
    }
  }
>
  test
</button>
`;
